oshi-core/src/main/java/oshi/hardware/IfOperStatus.java

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
There's an extra `*` here!   :-)

More seriously, though.  Since this `Enum` is associated very specifically with the `NetworkIF` interface, I'd like it as a nested class there, similar to the `State` enum in `OSProcess`.